lib/liquid.rb

```diff
@@ -24,6 +24,8 @@ module Liquid
   ArgumentSeparator           = ','
   FilterArgumentSeparator     = ':'
   VariableAttributeSeparator  = '.'
+  WhiteSpaceControl           = /-/
+  WhiteSpace                  = /[ \t]*\r?\n?[ \t]*/
```

Comment:
I don't think the newline should be optional.  For instance, in the strip_newlines filter we use `/\r?\n/`.

Also, what is the advantage of having it defined like this rather than just stripping all space characters preceding or following a tag using `/\s*/` like with go templates?